submissions/api.py

```diff
@@ -373,6 +383,40 @@ def get_submissions(student_item_dict, limit=None):
     return SubmissionSerializer(submission_models, many=True).data
 
 
+def get_all_submissions(course_id, item_id, item_type, read_replica=True):
```

Comment:
Also, note to self: Need to add in the code to actually enable the read replica for this query.